finesse/gui/logs_view.py

```diff
@@ -0,0 +1,48 @@
+"""Code for FINESSE's documentation viewer window."""
+import os
+import webbrowser
+
+from PySide6.QtCore import QObject, QUrl
+from PySide6.QtGui import QAction, QDesktopServices
+
+from finesse.logger import get_log_path
+
+
+class LogOpen(QAction):
+    """A menu option for opening the current log file."""
+
+    def __init__(self, parent: QObject) -> None:
+        """Create a menu item for opening the current log file.
+
+        Args:
+            parent: the menu on which to place the menu item
+        """
+        super().__init__("Open log", parent)
+        self.triggered.connect(self.open_log)
+
+    def open_log(self) -> None:
+        """Opens the current log file."""
+        self.path = os.path.realpath(get_log_path())
+        self.files = [
+            f"{self.path}/{x}" for x in os.listdir(self.path) if x.endswith(".log")
+        ]
+        self.log = max(self.files, key=os.path.getctime)
+        webbrowser.open(self.log)
```

Comment:
I think `Pathlib` is the way to go here to ensure cross-platform compatibility. 

However, I think we could do something cleaner than searching for the latest file. If we stored `filename` (defined in `initialise_logging()`) globally in `finesse.logger`, we could just import `finesse.logger.filename` and then we know it'll be the current session's log, and it'll be a `Path`, so will be OS-independent. You'll still need to pass it as a string to `webbrowser.open()`, but still. Anyone else (@dalonsoa @jamesturner246) have an opinion on this?

Reply:
Agree. `pathlib` is preferable, and covert to string only at the last moment.